README.md

```diff
@@ -539,30 +524,273 @@ services.Configure<TargetingEvaluationOptions>(options =>
+                        "Name": "Small",
+                        "ConfigurationReference": "ShoppingCart:Small",
+                        "AssignmentParameters": {
+                            "Audience": {
+                                "Users": [
+                                ],
+                                "Groups": [
+                                    {
+                                        "Name": "Ring1",
+                                        "RolloutPercentage": 50
+                                    }
+                                ],
+                                "DefaultRolloutPercentage": 30
+                            }
+                        }
+                    }
+                ]
+            }
+        }
+    },
+    "ShoppingCart": {
+        "Big": {
+            "Size": 400,
+            "Color": "green"
+        },
+        "Small": {
+            "Size": 150,
+            "Color": "gray"
+        }
+    }
+}
+```
+
+In the example above we see the declaration of a dynamic feature in a json configuration file. The dynamic feature is defined in the `FeatureManagement:DynamicFeatures` section of configuration. The name of this dynamic feature is `ShoppingCart`. A dynamic feature must declare a feature variant assigner that should be used to select a variant when requested. In this case the built-in `Targeting` feature variant assigner is used. The dynamic feature has two different variants that are available to the application. One variant is named `Big` and the other is named `Small`. Each variant contains a configuration reference denoted by the `ConfigurationReference` property. The configuration reference is a pointer to a section of application configuration that contains the options that should be used for that variant. The variant also contains assignment parameters denoted by the `AssignmentParameters` property. The assignment parameters are used by the assigner associated with the dynamic feature. The assigner reads the assignment parameters at run time when a variant of the dynamic feature is requested to choose which variant should be returned. 
+
+An application that is configured with this `ShoppingCart` dynamic feature may request the value of a variant of the feature at runtime through the use of `IDynamicFeatureManager.GetVariantAsync`. The dynamic feature uses targeting for [variant assignment](./README.md#Feature-Variant-Assignment) so each of the variants' assignment parameters specify a target audience that should receive the variant. For a walkthrough of how the targeting assigner would choose a variant in this scenario reference the [Microsoft.Targeting Assigner](./README.md#Microsoft.TargetingAssigner) section. When the feature manager chooses one of the variants it resolves the value of the variant by resolving the configuration reference declared in the variant. The example above includes the configuration that is referenced by the `ConfigurationReference` of each variant.

```

Comment:
The link for Microsoft.Targeting Assigner section should be: "./README.md#microsofttargeting-assigner"